src/components/ArticleCard.jsx

```diff
@@ -1,10 +1,64 @@
 import { useEffect, useState, useContext } from "react";
-import { Route, Routes, Link } from "react-router-dom";
+import { Route, Routes, Link, useParams } from "react-router-dom";
 import axios from "axios";
-import { getAllArticles } from "../utils";
+import { getAllArticles, getArticleById } from "../utils";
+import { getCommentsByArticleId } from "../utils";
 
-const ArticleCard = ({}) => {
-   
-}
+const ArticleCard = () => {
+  const { article_id } = useParams();
+  const [articlePage, setArticlePage] = useState([]);
+  const [comments, setComments] = useState([]);
+  useEffect(() => {
+    getArticleById(article_id)
+      .then(({ article }) => {
+        console.log(article, "<-- each article");
```

Comment:
Please remove console logs before merging. Console logs generally are not production code.